### ISSUE TITLE
Vertical Nav Bar: Accessibility Issue

### ISSUE THREAD

**Issue by kalpeshhpatel** [vmware/clarity#4117]
## Describe the bug

When using the collapsible navbar, name, and state of expand sidebar in the bottom of sidebar is not read by screen-reader.

1.  Go to 'https://v2.clarity.design/vertical-nav/collapsible-nav/normal'
2.  Click on the 'Collapse' sidebar button to collapse the sidebar.
3.  Screen reader correctly reads the name and state of the collapse/expand sidebar button using `aria-lablel` and `aria-expanded` attribute.
4.  However, when the sidebar is collapsed, there is also a button at the bottom to expand sidebar.

<img width="994" alt="Screen Shot 2019-12-16 at 11 38 18 AM" src="https://user-images.githubusercontent.com/5223493/70883458-1225b400-1ff9-11ea-8374-d4757af1127a.png">
 
The screen reader is not able to determine name and state of the button at the bottom of the sidebar.

## Expected behavior

The screen reader should be able to determine the name and state of the expand sidebar button which is present in the bottom too.

## Versions

This issue is reproducible with the latest clarity version.

**Comment by gnomeontherun** [comment]
We provide this button as a convenience to toggle by clicking on the bar.

@chlane If we have two buttons in the vertical nav bar (the actual toggle and the bottom is a duplicate to toggle by clicking on the bar), do they both need to be marked up for a screen reader, or perhaps it makes more sense to use `aria-hidden` on the second (duplicated) button?

**Comment by chlane** [comment]
If we are providing a button as a convenience we should do it universally IMO.              

aria-hidden="true" does not keep the browser from setting focus to a button. So screen reader users will hear nothing as the result of pressing the Tab key and setting browser focus to this element. While silence in itself is not that confusing, the interruption of announcements that were in progress could be confusing. This is what happens with NVDA/Firefox. See http://cx-guidelines.vmware.com/examples/ariaHiddenTrueOnButton.html.

I recommend fixing this issue.

**Comment by Shijir** [comment]
@chlane so the solution here is to add attributes to the button such as `aria-lablel` and `aria-expanded` attributes, right?

**Comment by chlane** [comment]
@Shijir Yes use aria-label and aria-expanded.

**Comment by Jinnie** [comment]
@chlane, @Shijir - can't we just get this "blank button area" outside of the tabbing order with tabindex="-1"?
It will still be clickable, to cover the "convinience scenario", but won't be confusing k/b users, blind users, to whom its presence is unnecessary.
Thinking a bit deeply - blind users should not care at all if the nav is expanded or collapsed. It will only pollute their SR reader with information that has no value to them and make the time needed to navigate even longer. We can save it to them.
tabindex -1 is better than aria-hidden, because there will be no "silence on navigation".
K/b only users will still have their button. To them, having a focusable blank ares will also be confusing IMO.

**Comment by bdryanovski** [comment]
This is related to the internal talk that we had with @Jinnie  that right now both buttons 
 * the one with the icon for hiding/expanding the vertical navigation
 * and the one that is there just to help people to expand the navigation 

Аre having `aria-label` that clearly says what is their purpose.

Also, something that I notice is that the second button is not clearly promoted as one, you could find it by mistake clicking on the navigation, hovering the navigation to see the cursor changing to a pointer or by tabbing - so maybe just removing it from the tab list is a better option.

**Comment by chlane** [comment]
@Shijir @Jinnie Ivan I think you are right. We should take a step further and hide it from screen reader cursors like we did with the Tabs Overflow ... button below.
`<li role="application" class="nav-item"><button type="button" aria-hidden="true" class="btn btn-link nav-link dropdown-toggle open" tabindex="-1">`

**Comment by chlane** [comment]
@bdryanovski Bozhidar removal would be the best
